controllers/users.js

```diff
@@ -607,6 +606,32 @@ const setInDiscordScript = async (req, res) => {
   }
 };
 
+/**
+ * Patch Update user nickname
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const updateUserNickname = async (req, res) => {
+  try {
+    const { user } = await userQuery.fetchUser({ userId: req.params.userId });
+
+    const { discordId, username: userName } = user;
+
+    if (!discordId) {
+      throw new Error("user not verified");
+    }
+
+    const response = await setUserDiscordNickname(userName, discordId);
+
+    return res.json({
+      message: response,
```

Comment:
add some better information in response rather than just sending back message, try making the response more user friendly.